api/v1alpha1/sysbench_types.go

```diff
@@ -33,9 +33,9 @@ type SysbenchSpec struct {
 	// +optional
 	PodConfig PodConfigurationSpec `json:"podConfig,inline"`
 
-	// Options is a list of zero or more command line options starting with '--'.
+	// Args is a list of zero or more command line arguments starting with '--'.
```

Comment:
Sysbench refers to this as common options. The above line was a quote from sysbench's readme:
https://github.com/akopytov/sysbench/blob/master/README.md

Do we want to change this? cc: @apollolemonte

Reply:
The reason I changed this was the original issue description, which included 'options'.